### DISCUSSION TITLE
How do i make Google Chrome window transparent?

### DISCUSSION THREAD

**Discussion by hansfigo** [hyprwm/Hyprland#2387] [Q&A]
Hey there, I'm having trouble making the Google Chrome window transparent in Hyprland using the code `windowrulev2 = opacity 0.8 0.8,class:^(google-chrome-stable)$ `Have you had any success with this? Any suggestions on how to make it work?

**Comment by ghost** [comment]
Hey, sorry for being late but you do realise that you have put 0.8 twice meaning that it will throw an error?

**Reply by vaxerski** [reply]
it won't, it's valid

**Reply by Cody-W-Tucker** [reply]
The line `"opacity 0.8 0.8,class:^(google-chrome)$"` sets the opacity for Google Chrome windows. The first `0.8` is the opacity level when the window is focused (active), and the second `0.8` is the opacity level when the window is unfocused (inactive).

**Reply by EvanSunde** [reply]
Lmao, Some people downvoted creator of hyprland eventhough he is right and he is the one who created it

**Comment by ghost** [comment]
Are you sure that is the correct class?

Please report `hyprctl clients` with Google Chrome opened

**Comment by cold-runner** [comment]
> 

windowrulev2 = opacity 0.8 0.8,class:^(google-chrome)$ 
will make it work

**Reply by cold-runner** [reply]
it should be google-chrome instead of google-chrome-stable

**Comment by hoarse-boy** [comment]
the `hyprctl clients` from that deleted user is needed, strangely mine has capital 'G' that's why it doesnt work with `google-chrome`.

```ini
windowrulev2 = opacity 0.80 0.80,class:^(Google-chrome)$
```

**Reply by MKamilAmin** [reply]
I had the same issue. It depends on the source of the chrome install. Using AUR I think it should be google-chrome-stable or just google-chrome. The Flatpak version (whitch I have) is "Google-chrome".
